README.md

```diff
@@ -177,7 +177,25 @@ for {
     ...
 }
 ```
+### Tailing from a specific time
 
+You can tail from a specific point in time.  For example:
+
+```go
+
+kinesisClient := ...
+s3Client := ...
+
+tail := triton.NewTailAt(&triton.NewTailAtParams{
```

Comment:
I think we need to rethink these 'public' interfaces.

Like perhaps the interfaces should all be "NewStreamReader"-ish with combinations for:

  * starting/maintaining a Checkpointer
  * starting at a specific Time